components/ai_chat/resources/untrusted_conversation_frame/components/assistant_task/assistant_task.module.scss

```diff
@@ -0,0 +1,82 @@
+.task {
+  border: 1px solid var(--leo-color-divider-subtle);
+  border-radius: var(--leo-radius-xl);
+  min-height: 350px;
+  box-shadow: var(--leo-effect-elevation-01);
+}
+
+.taskContent {
+  padding: var(--leo-spacing-2xl);
+  display: flex;
+  flex-direction: row;
+  gap: var(--leo-spacing-xs);
+  flex-wrap: wrap;
+}
+
+.taskData {
+  flex: 1 1 0;
+  min-width: min(300px, 100%);
+}
+
+.taskStep {
+  --task-icon-size: 22px;
+  --half-task-icon-size: calc(var(--task-icon-size) / 2);
+
+  min-height: 30px;
+  // Space to the left of the steps line to counteract the negative left position
+  // of the steps icon.
+  margin-left: var(--half-task-icon-size);
+  // The steps line
+  border-left: var(--leo-color-neutral-20) 2px solid;
+  // No padding-top so that the previous steps line extends to the top of this
+  // steps icon.
+  padding: 0 var(--leo-spacing-l) var(--leo-spacing-xl) var(--leo-spacing-l);
+  // Space to the right of the steps line
+  padding-left: calc(var(--half-task-icon-size) + var(--leo-spacing-m));
+
+  font: var(--leo-font-default-regular);
+  color: var(--leo-color-text-primary);
+  position: relative;
+
+  .taskIcon {
+    --border-size: 1px;
+    --leo-icon-size: 16px;
+    --leo-progressring-size: 16px;
+
+    position: absolute;
+    top: calc(0.5lh - var(--half-task-icon-size));
+    left: calc(var(--half-task-icon-size) * -1);
+    width: calc(var(--task-icon-size) - calc(2 * var(--border-size)));
+    height: calc(var(--task-icon-size) - calc(2 * var(--border-size)));
```

Comment:
just curious but any reason we can't use an outline here instead, so we don't need to worry about these size calculations?